src/main/java/com/micetr0/mock_DB/MySqlDB.java

```diff
@@ -258,38 +259,106 @@ public Boolean execute(Connection conn) throws SQLException {
 
 
     @Override
-    public void insertNote(Note note) {
+    public void insertNote(String type, String pitch, Integer measureIndex, Integer measureId) {
```

Comment:
Should return a Boolean

Reply:
actually should probably return a noteID, that would be more useful probably. Easy to change as we go if needed, but currently it returns noteID